src/main/webapp/support/index.jsp

```diff
@@ -37,7 +37,8 @@ the IRC chat room. You might just find someone ready to help on <em>#selenium at
 Freenode</em> (irc.freenode.net). There is more info about 
 <a href="http://en.wikipedia.org/wiki/Irc">IRC on wikipedia</a><br/>
 Today's <a href="javascript:(function(){var date=(function(){var d=new Date();d=new Date(d.getTime() + (d.getTimezoneOffset())*60000);m=d.getMonth()+1;if(m<10)m='0'+m;day=d.getDate();if(day<10)day='0'+day;return (d.getYear()+1900)+'/'+m+'/'+day;})();window.location='https://raw.github.com/SeleniumHQ/irc-logs/master/'+date+'.txt';})();">Selenium IRC Chat log</a>. You can bookmark this link to always get
-the current day's log.
+the current day's log. <strong>Note: </strong>IRC logs have not replicated to github
+since March 19, 2015.  An alternate archive is available at <a href="https://botbot.me/freenode/selenium/">botbot</a>.
```

Comment:
I like the idea of the alternate archive, but not sure how I feel about saying that they haven't been updated since March 19th.    

This might be an issue with the actual irc-logs repository script that logs it all.  

If it starts working tomorrow, we'd have to update this again.